lib/Graph/Nodes.cpp

```diff
@@ -2580,7 +2580,7 @@ bool MFCCNode::verify() const {
   isValid &= expectCompareTrue(
       "Number of coefficients should be smaller than the filter bank count",
```

Comment:
Actually just realized that the comment needs to be updated as well.

Reply:
Done